test/packages_test.rb

```diff
@@ -914,6 +914,168 @@ def product(properties = {})
     end
   end
 
+  describe "#check_remote_installation_packages" do
+    before do
+      allow(Yast::Linuxrc).to receive(:vnc).and_return vnc
+      allow(Yast::Linuxrc).to receive(:display_ip).and_return display_ip
+      allow(Yast::Linuxrc).to receive(:braille).and_return braille
+      allow(Yast::Linuxrc).to receive(:usessh).and_return usessh
+      allow(Yast::Packages).to receive(:vnc_packages).and_return(vnc_packages)
+      allow(Yast::Packages).to receive(:remote_x11_packages).and_return(remote_x11_packages)
+      allow(Yast::Packages).to receive(:ssh_packages).and_return(ssh_packages)
+      allow(Yast::Packages).to receive(:braille_packages).and_return(braille_packages)
+    end
+
+    let(:vnc_packages) { %w(some-vnc-packages) }
+    let(:remote_x11_packages) { %w(some-x11-packages) }
+    let(:ssh_packages) { %w(openssh iproute2) }
+    let(:braille_packages) { %w(sbl) }
+
+    context "on a boring local regular installation" do
```

Comment:
I do a lot of boring installations during testing... :stuck_out_tongue_closed_eyes: